### PR TITLE
move nimbledroid files to one place

### PR DESCRIPTION
**Pull request by klahnakoski**
move nimbledroid files to one directory
move redashFormatter.js into RedashContainer/index.jsx

### CODE REVIEW

**Comment by klahnakoski** [comment]
Partial solution to https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/288

File: src/containers/RedashContainer/index.jsx [hunk 1]
```diff
@@ -2,9 +2,55 @@ import React, { Component } from 'react';
...
+
+    return {
+      label: key,
+      ...generateDatasetStyle(SETTINGS.colors[index]),
+      data: dataToChartJSformat(datum.sort(sortByDate)),
+    };
+  });
+
+  return { datasets };
+};
```
Comment (klahnakoski) [inline src/containers/RedashContainer/index.jsx]: the above is from the file  https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/414/files#diff-729a1e388df9d305582838f2b94f0caf

File: src/nimbledroid/NimbledroidGraphContainer.jsx [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+    .map((details, packageId, index) => ({
+      data: selectFrom(details)
+        .filter(({ date }) => date > SINCE)
+        .select({ x: 'date', y: 'value' })
+        .toArray(),
+      label: CONFIG.packageIdLabels[packageId],
+      ...generateDatasetStyle(SETTINGS.colors[index]),
+    }))
+    .toArray(),
+});
```
Comment (klahnakoski) [inline src/nimbledroid/NimbledroidGraphContainer.jsx]: the above is from https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/414/files#diff-dcb7235400c9a406ffc7771c4b3e3c91

File: src/nimbledroid/config.js [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+  ],
+  summaryTable: [
+    'org.mozilla.klar',
+    'org.mozilla.geckoview_example',
+    'com.chrome.beta',
+  ],
+  baseProduct: 'org.mozilla.geckoview_example',
+  compareProduct: 'com.chrome.beta',
+  targetRatio: 1.2,
+};
```
Comment (klahnakoski) [inline src/nimbledroid/config.js]: the above is from https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/414/files#diff-12464aba5618f7693c828c892396a8db

**Comment by armenzg** [comment]
I won't be able to review this until I come back from parental leave.
You can find another reviewer or once I have time.

**Comment by klahnakoski** [comment]
@armenzg 

Yes, please review when you get back.  Even though it will be merged by then, I can still use you feedback.

**Comment by karlht** [comment]
At a superficial level, this looks all right to me.

**Pull request merged**